x/staging/subspaces/client/cli/tx.go

```diff
@@ -0,0 +1,249 @@
+package cli
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/version"
+	"github.com/desmos-labs/desmos/x/staging/subspaces/types"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+// NewTxCmd returns a new command to perform subspaces transactions
+func NewTxCmd() *cobra.Command {
+	subspacesTxCmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      "Subspaces transaction subcommands",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	subspacesTxCmd.AddCommand(
+		GetCmdCreateSubspace(),
+		GetCmdAddSubspaceAdmin(),
+		GetCmdRemoveSubspaceAdmin(),
+		GetCmdRegisterUser(),
+		GetCmdBlockUser(),
+		GetCmdEditSubspace(),
+	)
+
+	return subspacesTxCmd
+}
+
+// GetCmdCreateSubspace returns the command used to create a subspace
+func GetCmdCreateSubspace() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create [subspace-id] [name] [[open]]",
+		Args:  cobra.RangeArgs(2, 3),
+		Short: "Create a subspace with the given [subspace-id], [name] and [open]",
```

Comment:
No need to specify all the flags
```suggestion
		Short: "Create a new subspace",
```